modules/scripts.py

```diff
@@ -529,10 +530,17 @@ def postprocess(self, p, processed):
                 errors.report(f"Error running postprocess: {script.filename}", exc_info=True)
 
     def postprocess_batch(self, p, images, **kwargs):
+        image_list = kwargs.get('image_list', None)
+
         for script in self.alwayson_scripts:
             try:
                 script_args = p.script_args[script.args_from:script.args_to]
                 script.postprocess_batch(p, *script_args, images=images, **kwargs)
+
+                # make images and image_list share the same storage
+                if image_list is not None:
+                    images = torch.stack(image_list) if image_list else torch.tensor([]).reshape(0, 3, p.height, p.width)

```

Comment:
This doesn't work if all images do not have the same shape. I'll seek another solution.